torch/nn/modules/activation.py

```diff
@@ -646,6 +646,12 @@ class GELU(Module):
 
     where :math:`\Phi(x)` is the Cumulative Distribution Function for Gaussian Distribution.
 
+    When the approximate flag is enabled, Gelu is estimated with:
+        :math::  \text{GELU}(x) = 0.5 * x * (1 + \text{Tanh}(\sqrt(2 / \pi) * (x + 0.044715 * x^3)))
```

Comment:
It should be good to add references to models / codebases that use this particular approximation, e.g. refer to original google bert impl (taking account that there are two different approximations in the wild)